<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>306</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/setting.png</normaloff>:/icons/icons/setting.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>房间价格</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>32</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>标准间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>65</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>双人间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>98</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>大床间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>131</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>商务间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>164</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VIP间</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>22</y>
      <width>135</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditRoom1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRoom2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRoom3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRoom4"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditRoom5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>房间消费价格</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>32</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>饮用水</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>65</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>饼干</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>98</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>即食面</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>131</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>蔬果拼盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>164</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>早餐</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>22</y>
      <width>135</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditConsume1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConsume2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConsume3"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConsume4"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditConsume5"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBKG">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>391</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>系统设置</string>
   </property>
   <widget class="QCheckBox" name="checkBoxSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>模糊搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时间间隔：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>百毫秒</string>
    </property>
   </widget>
  </widget>
  <zorder>labelBKG</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>okButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>resetButton</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>lineEditRoom1</tabstop>
  <tabstop>lineEditRoom2</tabstop>
  <tabstop>lineEditRoom3</tabstop>
  <tabstop>lineEditRoom4</tabstop>
  <tabstop>lineEditRoom5</tabstop>
  <tabstop>lineEditConsume1</tabstop>
  <tabstop>lineEditConsume2</tabstop>
  <tabstop>lineEditConsume3</tabstop>
  <tabstop>lineEditConsume4</tabstop>
  <tabstop>lineEditConsume5</tabstop>
  <tabstop>checkBoxSearch</tabstop>
  <tabstop>lineEditTime</tabstop>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
